tmol/score/dunbrack/potentials/dunbrack_pose_score.impl.hh

```diff
@@ -363,7 +373,339 @@ auto DunbrackPoseScoreDispatch<DeviceDispatch, D, Real, Int>::f(
 
   DeviceDispatch<D>::forall_stacks(n_poses, max_n_blocks, func);
 
+  printf("FORWARD SUCCESS\n");
+
   return {V_t, dV_dx_t};
+}
+
+template <
+    template <tmol::Device>
+    class DeviceDispatch,
+    tmol::Device D,
+    typename Real,
+    typename Int>
+auto DunbrackPoseScoreDispatch<DeviceDispatch, D, Real, Int>::backward(
```

Comment:
so this is fine as is.  But for score terms like dunbrack that only put terms on the diagonal, it may be possible to use stored derivs instead?  I think we should keep this scheme for now however.